pacman/executor/pacman_algorithm_executor.py

```diff
@@ -432,7 +431,8 @@ def _determine_algorithm_order(
                     "    Tokens complete: {}\n"
                     "    Fake tokens complete: {}\n"
                     "    Tokens to find: {}\n"
-                    "    Functions available: {}\n"
+                    "    Algorithms Left: {}\n"
```

Comment:
It might be better to label this `Required algorithms remaining to be used:` or something like that.

Reply:
Agreed todo